tests/e2e/Services/Account/AccountBase.php

```diff
@@ -931,6 +932,58 @@ public function testUpdateAccountVerification($data): array
         return $data;
     }
 
+    /**
+     * @depends testUpdateAccountVerification
+     */
+    public function testCreateAccountVerificationForVerifiedEmail($data): array
+    {
+        $email = $data['email'] ?? '';
+        $name = $data['name'] ?? '';
+        $session = $data['session'] ?? '';
+
+        /**
+         * Test for SUCCESS
+         */
+        $response = $this->client->call(Client::METHOD_POST, '/account/verification', array_merge([
+            'origin' => 'http://localhost',
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'cookie' => 'a_session_' . $this->getProject()['$id'] . '=' . $session,
+
+        ]), [
+            'url' => 'http://localhost/verification',
+        ]);
+
+        $this->assertEquals(409, $response['headers']['status-code']);
+
+        /**
+         * Test for FAILURE
+         */
+        $response = $this->client->call(Client::METHOD_POST, '/account/verification', array_merge([
+            'origin' => 'http://localhost',
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'cookie' => 'a_session_' . $this->getProject()['$id'] . '=' . $session,
+        ]), [
+            'url' => 'localhost/verification',
+        ]);
+
+        $this->assertEquals(400, $response['headers']['status-code']);
```

Comment:
This is an error because the url is invalid? We have this test elsewhere so no need to do that here.